server/services/index.ts

```diff
@@ -9,6 +9,7 @@ import CommunityOffenderManagerCaseloadService from './communityOffenderManagerC
 import UserService from './userService'
 import ResidentialChecksService from './residentialChecksService'
 import DecisionMakerCaseloadService from './decisionMakerCaseloadService'
+import PdfService from './pdfService'
```

Comment:
Maybe we could rename some of the references to pdf to forms? 

`FormService`
`AssessmentFormRoutes`
`getForm`
`renderPdf`